File: Education/TPRecycler/app/src/main/java/com/example/tprecycler/MyViewHolder.java
package com.example.tprecycler;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;


public class MyViewHolder extends RecyclerView.ViewHolder
{
    TextView textViewName;
    ImageView imageViewIcon;
    public MyViewHolder(View itemView) {
        super(itemView);
        this.textViewName=itemView.findViewById(R.id.tit);
        this.imageViewIcon=itemView.findViewById(R.id.img);
    }
}

File: MyApplication/FirstAppKotlin/app/src/main/java/com/example/firstappkotlin/MyViewHolder.kt
package com.example.firstappkotlin
import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)

{ val color: TextView=itemView.findViewById(R.id.lig)
}